build.gradle

```diff
@@ -59,12 +59,12 @@ repositories {
 
 dependencies {
     implementation "org.gradle:gradle-tooling-api:+"
-    api 'gradle.plugin.net.wooga.gradle:atlas-unity:2.0.0-rc.6'
-    api 'gradle.plugin.net.wooga.gradle:atlas-secrets:1.0.0-rc.1'
+    api 'gradle.plugin.net.wooga.gradle:atlas-unity:[2,3)'
+    api 'gradle.plugin.net.wooga.gradle:atlas-secrets:[1,2)'
```

Comment:
there is not yet a final release yet correct? As long as this works we can do it.

Reply:
For which? atlas-unity had a release yesterday.